exercises/resistor-color-duo/README.md

```diff
@@ -6,9 +6,9 @@ If you want to build something using a Raspberry Pi, you'll probably use _resist
 * Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
 To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band acts as a digit of a number. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
 
-In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take two colors as input, and output the correct number.
+In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take two colors as input, and output the resistance value.
```

Comment:
This wording may need to be changed in exercism/problem-specifications instead of on the track... I don't think it is specific to the track.

Reply:
If someone confirms for sure that's what needs to be done and that we want this change I can do that.

Reply:
As the description is not track-specific, it needs to be done there ultimately. That said, I'm working on rewriting the description entirely. Are you okay with waiting for that? I'll make sure to ping you. 
I'd like to wait until Jeremy is back before discussing it, because it's his description, and it's kind of an exemplar of what the product team wants in (future) exercise descriptions. 
You okay with this?

Reply:
Sure, it can wait a bit.

Reply:
PS First things first: I will definitively include the removal of 'encoding' .

Reply:
GOOD.